src/screens/doc/DocIconsScreen.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+
+import { RegularText } from '../../components';
+import { colors, Icon, normalize } from '../../config';
+
+type TIconItem = {
+  iconName: keyof typeof Icon;
+};
+
+const IconItem = ({ iconName }: TIconItem) => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const SelectedIcon: any = Icon[iconName];
+
+  return (
+    <View style={styles.iconContainer}>
+      <SelectedIcon />
```

Comment:
how come that you have added the ts lint ignore comment for line 13 and not for 17? i thought line 17 was making the issue?
why is `: any` added in line 13? i think it wasn't there before?

Reply:
with the type `: any` the ts error for `SelectedIcon` was solved but a eslint ts error occured with for `any`, therefore the ts lint ignore comment for line 13, for the type `any`

Reply:
so there is an `any` added that needs to be ignored to fix the initial ts issue? why not ignoring the initial complain directly?

Reply:
cause it was a ts issue not a es lint issue, I haven't found a way and neither did Arda to ignore a ts issue. Therefore we "fixed" it with giving it a type and ignored the type issue of `any` which was now a es lint issue that could be disabled via the eslint-disable comment

Reply:
ok but i understand any type issue as TS as well?
we already have ts error comments in our code, please have a look for `// @ts-expect-error`s, which would be a better case here instead if working around.

Reply:
thank you for the tip, updated the code accordingly.
https://github.com/smart-village-solutions/smart-village-app-app/pull/914/commits/281ea8084181c28f9fec438f40df3491f44e3057